materials/tidy-data.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: page
+element: notes
+title: Tidy Data
+language: SQL
+---
+
+The main goal of tidy data is making it easy for a computer to work with the
+data. Let's start by looking at some messy data and thinking about what makes it
+messy and what we could do to improve it.
+
+> Do the exercise on [Improving Messy Data]({{ site.baseurl }}/exercises/Tidy-data-improving-messy-data-SQL).
+
```

Comment:
A super minor point...

You have a blank line in between the `> Do action` and the points that you hope to cover during that action. I place a `>` in that blank line to keep the block quote bar continuous (one action).